instance/models/server.py

```diff
@@ -29,6 +29,7 @@
 from django.db import models
 from django.db.models import Q
 from django_extensions.db.models import TimeStampedModel
+from django_extensions.db.fields.json import JSONField
```

Comment:
For new fields, we should be using `from django.contrib.postgres.fields import JSONField`. The `django_extensions` version is deprecated in favor of the django one, and the native django one used the postgres JSONB type, which is much more efficient and powerful than the TEXT type used by this. I intend to migrate all the code in opencraft IM to use the new JSON field type at some point.